### PR TITLE
TE-372 Update readme

### PR DESCRIPTION
**Pull request by Jonnygwi**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-372)

### What **one** thing does this PR do?
Updates the Readme to be more consumer facing not contributor facing

### Any other notes
image

### CODE REVIEW

**Comment by Jonnygwi** [comment]
Still need to add 'shields' when we have an NPM package

File: Readme.md [hunk 1]
```diff
@@ -1,164 +1,73 @@
...
-
-```bash
-$ git clone git@github.com:lodgify/lodgify-ui.git
-$ cd lodgify-ui
-$ npm install
-$ npm start
-```
-
-Then go to http://localhost:6060/
+<html>
```
Comment (marcdavi-es) [inline Readme.md]: We don't need the `<html>` tags.

Comment (Jonnygwi) [reply Readme.md]: It's not necessary for the Github md preview but I'm not sure how other services will render it. For example http://www.npmjs.com/

Comment (marcdavi-es) [reply Readme.md]: Pretty sure they're just standard markdown parsers

File: Readme.md [hunk 2]
```diff
@@ -1,164 +1,69 @@
...
+  <a href="https://lodgify.github.io/lodgify-ui/">
+    <img src="https://li5.cdbcdn.com/oh/a84645c0-0a35-4735-bda5-e5c425c2ffdc.png" width="150" />
+  </a>  
 
-React components for building amazing websites with Lodgify. https://lodgify.github.io/lodgify-ui/
+  **React components for building amazing websites with Lodgify.**  
 
-### Quick start
+  ![Travis](https://img.shields.io/travis/lodgify/lodgify-ui.svg?style=flat-square)
+  ![David](https://img.shields.io/david/lodgify/lodgify-ui.svg?style=flat-square)
```
Comment (davidpelayo) [inline Readme.md]: This is how this looks like currently
screen shot 2018-05-14 at 11 34 09 am

**Pull request merged**